src/test/java/com/puppycrawl/tools/checkstyle/api/ScopeTest.java

```diff
@@ -74,15 +74,15 @@ public void testMixedCaseSpaces() {
 
     @Test
     public void testMixedCaseSpacesWithDifferentLocales() {
-        final Locale[] differentLocales = {new Locale("TR", "tr") };
-        final Locale defaultLocale = Locale.getDefault();
         try {
+            final Locale[] differentLocales = {new Locale("TR", "tr") };
             for (Locale differentLocale : differentLocales) {
                 Locale.setDefault(differentLocale);
                 testMixedCaseSpaces();
             }
         }
         finally {
+            final Locale defaultLocale = Locale.getDefault();
```

Comment:
This actually breaks the test.
Purpose of variable outside the block was to save it's original state before we start changing it inside the `try`.
default locale at this point will now always be `turkish` instead of whatever the original was.